mlos_bench/mlos_bench/optimizers/base_optimizer.py

```diff
@@ -186,7 +185,7 @@ def config_space(self) -> ConfigurationSpace:
 
         Returns
         -------
-        ConfigurationSpace
+        ConfigSpace.ConfigurationSpace
```

Comment:
same issue as with the `datetime` above. we import `from ConfigSpace import ConfigurationSpace` so I would expect we use just
```suggestion
        ConfigurationSpace
```
here